src/Compilers/Test/Utilities/CSharp/TestOptions.cs

```diff
@@ -83,34 +81,6 @@ public static CSharpParseOptions WithReplaceFeature(this CSharpParseOptions opti
         {
             return options;
         }
-
-        internal static CSharpParseOptions WithExperimental(this CSharpParseOptions options, params MessageID[] features)
```

Comment:
Please do not remove this from code from the compiler. It is intended to facilitate experimental code residing in branches of the compiler, even though the product code does not contain experiments.